### PR TITLE
sci-mathematics/gismo: added ebuild for gismo (Geometry+Simulation)

### PR DESCRIPTION
**Pull request by timeraider4u**
added ebuild for gismo (Geometry+Simulation),
see https://gs.jku.at/gismo for more information on the project

### CODE REVIEW

File: sci-mathematics/gismo/gismo-0.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils eutils
+
+DESCRIPTION='G+Smo is a C++ library for geometric design and numerical simulation.'
```
Comment (jlec) [inline sci-mathematics/gismo/gismo-0.8.1.ebuild]: Make it shorter like
```DESCRIPTION='C++ library for geometric design and numerical simulation'```

File: sci-mathematics/gismo/gismo-0.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND=""
+
+# Unbundling in progress, 
+# preparing local changes to get upstream
+DEPEND="${RDEPEND}"
```
Comment (jlec) [inline sci-mathematics/gismo/gismo-0.8.1.ebuild]: Empty *DEPEND can be dropped.

File: sci-mathematics/gismo/gismo-0.8.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+CMAKE_BUILD_TYPE="Release"
```
Comment (jlec) [inline sci-mathematics/gismo/gismo-0.8.1.ebuild]: Why not ```Gentoo```?

**Comment by timeraider4u** [comment]
@jlec: Thank you for your feedback. Seems that I have not read the section about CMAKE_BUILD_TYPE carefully enough ;-).

**Comment by jlec** [comment]
> Seems that I have not read the section about CMAKE_BUILD_TYPE carefully enough
Sometimes build breaks when we set something upstream doesn't expect. Just wanted to know if thatÄs the case here.

File: sci-mathematics/gismo/gismo-0.8.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples"
+
+# Unbundling in progress, 
+# preparing local changes to get upstream
+
+
```
Comment (jlec) [inline sci-mathematics/gismo/gismo-0.8.1.ebuild]: multiple empty lines. Are you using repoman? it should have been complaining about that.

File: sci-mathematics/gismo/gismo-0.8.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+src_configure() {
+	local mycmakeargs=(
+		$(cmake-utils_use examples GISMO_BUILD_EXAMPLES)
+	)
+	cmake-utils_src_configure
+}
+
+src_compile() {
+	cmake-utils_src_compile
+	cd "${S}_build"
```
Comment (jlec) [inline sci-mathematics/gismo/gismo-0.8.1.ebuild]: missing ```die```. All external commands need to have a ```|| die```

**Comment by timeraider4u** [comment]
Thanks for your replies. There are only some optimizations done when running in Release mode (CPU architecture, etc.), but I have seen that they are all already handled by portage itself.

**Comment by jlec** [comment]
Thanks for contributions.

**Comment by timeraider4u** [comment]
Thank you very much for your feedback and the fast merge ;-).

**Pull request merged**